sdapi_ingest_scripts/one_off/jobs/add_works_to_collections_job.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class AddWorksToCollectionsJob < ApplicationJob
+  queue_as :relationships
+
+  retry_on StandardError, attempts: 0
+
+  def perform
+    csv_path = "lib/data/articles_parent_child_relationships_final.csv"
```

Comment:
Why not put the csv data into `sdapi_ingest_scripts/data` and adjust the path?

Reply:
sure, can do